libs/domain/feature-loaders/src/snapshot/SnapshotProposalLoader.ts

```diff
@@ -0,0 +1,187 @@
+import { createGraphQLClient, GraphQLClient, ProgramError } from "@banklessdao/util-data";
+import { InitContext, InternalLoadContext, LoadContext, ScheduleMode } from "@shared/util-loaders";
+import { Data, Nested, NonEmptyProperty, OptionalNumberArrayOf, OptionalObjectRef, OptionalProperty, OptionalStringArrayOf, Property, RequiredArrayRef, RequiredStringArrayOf } from "@banklessdao/util-schema";
+import { DocumentNode } from "graphql";
+import gql from "graphql-tag";
+import * as t from "io-ts";
+import * as TE from "fp-ts/TaskEither";
+import { GraphQLDataLoaderBase } from "../base/GraphQLDataLoaderBase";
+import { BATCH_SIZE } from "../defaults";
+import { withMessage } from "io-ts-types";
+
+export const URL = "https://hub.snapshot.org/graphql";
+
+export const makeQUERY: (space:String)=>DocumentNode = (space) => {
+    space = `"${space}"` // So it gets parsed correctly
+    return gql`
+        query snapshotProposals($limit: Int,$cursor: Int) {
+            proposals (
+                first: $limit,
+                where: {
+                    space_in: [${space}],
```

Comment:
I haven't checked the docs of the `gql` function but I have a feeling that this line exposes a vulnerability to an injection attack. Can you double-check their docs that this is safe to do?